doc/fwutil/fwutil.md

```diff
@@ -252,35 +271,69 @@ Chassis1           BIOS        0ACLH004_02.02.007  Chassis BIOS
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required  Cold rebot, Fast reboot
+```
+
+3. Modular chassis platform
 ```bash
 root@sonic:~# fwutil show updates --image=next
-Chassis   Module   Component   Firmware               Version (current/available)              Status
---------  -------  ----------  ---------------------  ---------------------------------------  ------------------
-Chassis1           BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
-                   CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
-                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
-          Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
-                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+Chassis   Module   Component   Firmware               Version (current/available)              Status              Required Boot Action
+--------  -------  ----------  ---------------------  ---------------------------------------  ------------------  -----------------------
+Chassis1           BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required  N/A
+                   CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  N/A
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date          N/A
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required  N/A
+          Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  N/A
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date          N/A
+```
+
+4. Custom FW Package when the "boot" key and value are available in `platform_components.json`
+```bash
+root@sonic:~# fwutil show updates --fw-image=fw_update.tar.gz
+Component   Firmware               Version (current/available)              Status              Required Boot Action
+----------  ---------------------  ---------------------------------------  ------------------  -----------------------
+BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / NA                  update is required  None
+CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  Power cycle
+FPGA        <image_path>/fpga.bin  5 / NA                                   up-to-date          Cold reboot
+SSD         <image_path>/ssd.bin   4 / 5                                    update is required  Cold reboot, Fast reboot
+```
+
+5. Auto-update status
+```bash
+root@sonic:~# fwutil show auto-updates status
```

Comment:
@sujinmkang it seems that you have forgotten to add `auto-updates` to `show` command section tree

Reply:
@nazariig I will update.